internal/k8s/k8s.go

```diff
@@ -214,6 +214,9 @@ func (c *Client) Errorf(svc *v1.Service, kind, msg string, args ...interface{})
 	c.events.Eventf(svc, v1.EventTypeWarning, kind, msg, args...)
 }
 
+// ClientCoreV1 returns the embeded k8s client from Client.
+func (c *Client) ClientCoreV1() v1core.CoreV1Interface { return c.client.CoreV1() }
```

Comment:
I'm not a fan of this, because it leaks the whole k8s API to all clients. That'll make it significantly harder to test arp-speaker without having to deal with k8s machinery.

Maybe some time later, move the k8s-ish leader election code into internal/k8s, so we can keep the k8s-specific implementation bits hidden?

But as an incremental step, this is fine, no action required in this PR.